setup.py

```diff
@@ -33,12 +33,14 @@
     'tutorial': ['jupyter', 'matplotlib'],
 }
 
+GMS_EXT = ['.gms', '.opt', '.md']
+
 
 def all_subdirs(path, strip=None):
```

Comment:
should rename this function if the behavior is changing

Reply:
renamed to `all_gams_files()`